ci-operator/config/openshift/gcp-filestore-csi-driver-operator/openshift-gcp-filestore-csi-driver-operator-main.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+binary_build_commands: make
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+promotion:
+  name: "4.12"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: make test
+tests:
+- as: unit
+  commands: make test-unit
```

Comment:
This is failing because there's no Makefile yet.
Do you want to remove this to get tide/prow setup first? Or add a basic Makefile first and retest this PR?

Reply:
What about merging this as is and adding the Makefile (and the rest of the operator code) later?

Reply:
I can submit the PR right after this merges.